com.mimacom.ddd.sm.sus/META-INF/MANIFEST.MF

```diff
@@ -1,27 +1,29 @@
 Manifest-Version: 1.0
 Automatic-Module-Name: com.mimacom.ddd.sm.sus
 Bundle-ManifestVersion: 2
-Bundle-Name: com.mimacom.ddd.sm.sus
-Bundle-Vendor: mimacom ag
+Bundle-Name: %pluginName
+Bundle-Vendor: %providerName
```

Comment:
Same issue regarding name in generated product:

<img width="711" alt="image" src="https://user-images.githubusercontent.com/1107810/79303092-5a44dc80-7eee-11ea-90d1-948a99b3ab8f.png">